pkg/contexts/oci/testhelper/manifests.go

```diff
@@ -42,7 +42,8 @@ func HashManifest1(fmt string) string {
 	if fmt == artifactset.FORMAT_OCI || fmt == artifactset.OCIArtifactSetDescriptorFileName {
 		// hash = "sha256:334b587868e607fe2ce74c27d7f75e90b6391fe91b808b2d42ad1bfcc5651a66"
 		// hash = "sha256:0a326cc646d24f48c9bc79d303f7626404d41f2646934ef713cd1917bd5480ce" // with gardener.cloud legacy format
-		hash = "sha256:fafabfc2f9861c2ecf0ee3fc584ef4fb92c927902c8f561f72542281097cff83"
+		//hash = "sha256:fafabfc2f9861c2ecf0ee3fc584ef4fb92c927902c8f561f72542281097cff83"
+		hash = "sha256:b0692bcec00e0a875b6b280f3209d6776f3eca128adcb7e81e82fd32127c0c62"
```

Comment:
@ jensh007 Would you happen to know what this is and why I need to update it? :D

Reply:
I finally found the reason. When an OCI image is transferred by-value, it is transformed into a local artifact blob, which is a gzipped OCI distribution archive. What is changing now is the digest of this archive, Although the content and the meta data (timestamps) are still exactly the same, the compressed byte-stream changes after the included version update.
The reason seems to be an update of the `hauspost/compression` package., which leads to a change the compresses byte stream for the same initial content. 

I found another lib, which looks more standard (`compress/gzip`). When using this one, yet another compressed content is generated for the same initial uncompressed tar archive.

Fortunately the signing relevant digests for an OCI artefact package are NOT calculated from the byte-stream, but by a digest handler from the packaged content (here the manifest). Therefore signing over transport steps (changing the storage format) should still work.

Nevertheless we should decide on the gzip package to use. Any preferences?

Reply:
Ah, I see. If we can, we should switch to compress/gzip. Can we do that in a separate PR without further complicating this one? :)

Reply:
Just one more thought: Would it make sense to use a component-descriptor as base for the installation instead of the URL? It would give us the possibility to transfer artifacts to a private environment and install from there. A certain benefit of what we try solve with ocm?

Reply:
Indeed, that will be the next time. :) With bootstrapping and keeping the controller up-to-date, we'll switch to using a component. This is the basis of the incremental process.